File: 03.API/src/models/OrderDetail.js
const { DataTypes } = require('sequelize');
const sequelize = require('../config/database');

const OrderDetail = sequelize.define('OrderDetail', {
  id_detail: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
  id_order: { type: DataTypes.INTEGER, allowNull: false },
  id_product: { type: DataTypes.INTEGER, allowNull: false },
  quantity_detail: { type: DataTypes.INTEGER, defaultValue: 1 },
  price_detail: { type: DataTypes.FLOAT, defaultValue: 0 }
}, {
  tableName: 'order_details',
  timestamps: false
});

module.exports = OrderDetail;

File: 01.App/app/src/main/java/com/example/dacnapp/data/model/order/OrderDetail.java
package com.example.dacnapp.data.model.order;
import com.example.dacnapp.data.model.product.Product;

public class OrderDetail {
    public int id_detail;
    public int id_order;
    public int id_product;
    public int quantity_detail;
    public float price_detail;
    public Product Product;
}
